packages/web/components/trade-clipboard/index.tsx

```diff
@@ -850,9 +857,22 @@ export const TradeClipboard: FunctionComponent<{
 
           <div
             className={classNames(
-              "relative rounded-lg bg-osmoverse-900 px-4 md:px-3 transition-all ease-inOutBack duration-300 overflow-hidden",
-              showEstimateDetails ? "h-56 py-6" : "h-11 py-[10px]"
+              "relative rounded-lg bg-osmoverse-900 px-4 md:px-3 transition-all ease-inOutBack duration-300 overflow-hidden"
             )}
+            style={{
```

Comment:
I'd prefer to only use style as last result.

You can use an object as a parameter of `classNames` function above to conditionally apply utility classes. See [this](https://tailwindcss.com/docs/padding#arbitrary-values) on how to assign arbitrary values in tailwind.

Also, IMO these const values are unneeded and clutter the file. IMO here it's fine to just hardcode the values. Const defs like these are more for highly-consequential hardcoded values, i.e. how many cards to display in the view.

Reply:
Sounds good. I also don't like to lean into using style given its performance. I used it since can't use variable interpolation in Tailwind, and we need to calculate the height of the content dynamically.  

I'll remove the const values